bpfd-operator/apis/v1alpha1/shared_types.go

```diff
@@ -102,3 +102,178 @@ type ImagePullSecretSelector struct {
 	// Namespace of the secret which contains the credentials to access the image repository.
 	Namespace string `json:"namespace"`
 }
+
+// -----------------------------------------------------------------------------
+// Status Conditions - BPF Programs
+// -----------------------------------------------------------------------------
+
+const (
+	// BpfdOperatorFinalizer is the finalizer that holds a BPF program from
+	// deletion until cleanup can be performed.
+	BpfdOperatorFinalizer = "bpfd.io.operator/finalizer"
+)
+
+// -----------------------------------------------------------------------------
+// Status Conditions - BPF Programs
+// -----------------------------------------------------------------------------
+
+// ProgramConditionType is a condition type to indicate the status of a BPF
+// program at the cluster level.
+type ProgramConditionType string
+
+const (
+	// BpfProgConfigNotYetLoaded indicates that the program in question has not
+	// yet been loaded on all nodes in the cluster.
+	BpfProgConfigNotYetLoaded ProgramConditionType = "NotYetLoaded"
```

Comment:
```suggestion
	ProgramNotYetLoaded ProgramConditionType = "NotYetLoaded"
```

I know we said this was just to move the conditions but can we go ahead and at least rename this to remove all the references to the old `BpfProgramConfig` CRD?

Reply:
Sounds good, all of them were renamed in 209dd13c23f67192a258120202706c56fa838c55